lib/single-new-eks-opensource-observability-pattern/index.ts

```diff
@@ -38,9 +39,21 @@ export default class SingleNewEksOpenSourceobservabilityPattern {
             }
         };
 
+        const jsonString = fs.readFileSync(__dirname + '/../../cdk.json', 'utf-8');
+        const jsonStringnew = JSON.parse(jsonString);
+        let doc = utils.readYamlDocument(__dirname + '/../common/resources/otel-collector-config.yml');
```

Comment:
Why do we need this? Since the AmpAddon already has this logic for `enableAPIserverJob` flag and you are also passing the `otel-collector-config.yml` so will the AmpAddon not take care of this logic ? 
@freschri Please check this with Naveen.